docs/api/download-item.md

```diff
@@ -100,6 +100,8 @@ Returns `Boolean` - Whether the download is paused.
 
 Resumes the download that has been paused.
 
+**Note:** To enable resumable downloads in the narrow sense the server you are downloading from must support range requests and provide a Last-Modified as well as an ETag header value. Otherwise `resume()` will dismiss previously received bytes and restart the download from the beginning. 
```

Comment:
Thanks for adding this, it is very helpful, my only feedback is I'm not sure what `in the narrow sense` means in this context.

Removing it still reads fine to me, what were you trying to convey with it?